tracer/src/Datadog.Tracer.Native/iast/dataflow.h

```diff
@@ -66,6 +66,10 @@ namespace iast
         std::vector<WSTRING> _methodAttributeIncludeFilters;
         std::vector<WSTRING> _methodAttributeExcludeFilters;
 
+        int _callsiteInstrumentedSources = 0;
```

Comment:
I'd rather maintain just an array with all the metrics. Sources have no type now, but that could change. We could set a flag for the aspect type (0x10000000 propagation, 0x20000000 source, 0x40000000 sink or something similar) and the type in the lower part of the index value. This way, we could be more future proof when reporting the metrics.